diff -urN serf-1.3.9/SConstruct serf-1.3.9-patched/SConstruct
--- serf-1.3.9/SConstruct	2015-09-17 05:46:24.000000000 -0700
+++ serf-1.3.9-patched/SConstruct	2018-04-25 17:52:36.449269037 -0700
@@ -55,7 +55,7 @@
 # To be used to ensure a PREFIX directory is only created when installing.
 def createPathIsDirCreateWithTarget(target):
   def my_validator(key, val, env):
-    build_targets = (map(str, BUILD_TARGETS))
+    build_targets = (list(map(str, BUILD_TARGETS)))
     if target in build_targets:
       return PathVariable.PathIsDirCreate(key, val, env)
     else:
@@ -183,7 +183,7 @@
 
 unknown = opts.UnknownVariables()
 if unknown:
-  print('Warning: Used unknown variables:', ', '.join(unknown.keys()))
+  print('Warning: Used unknown variables:', ', '.join(list(unknown.keys())))
 
 apr = str(env['APR'])
 apu = str(env['APU'])
